<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CADRE</class>
 <widget class="QMainWindow" name="C_CADRE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_CADRE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat Carte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat Serveur</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Etat_Carte">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="Etat_Serveur">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="Info_Serveur">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="Info_Carte">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Connection_Serveur">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Connexion_Carte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>282</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Bouton_Connection_Serveur</sender>
   <signal>clicked()</signal>
   <receiver>C_CADRE</receiver>
   <slot>Slot_Connexion_Serveur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bouton_Connexion_Carte</sender>
   <signal>clicked()</signal>
   <receiver>C_CADRE</receiver>
   <slot>Slot_Connexion_Carte()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Slot_Connexion_Serveur()</slot>
  <slot>Slot_Connexion_Carte()</slot>
 </slots>
</ui>
